docs/lua-api/classes/uworld.md

```diff
@@ -0,0 +1,11 @@
+# UWorld
+## Inheritance
+[RemoteObject](./remoteobject.md)
+
+## Methods
+
+### SpawnActor(UClass ActorClass, FVector Location, FRotator Rotation)
+It's a self-implemented function that is automatically added to every UWorld object by UE4SS.  
+The function uses `UGameplayStatics:BeginDeferredActorSpawnFromClass` and `UGameplayStatics:FinishSpawningActor` to spawn an actor.  
```

Comment:
```suggestion
This function uses `UGameplayStatics:BeginDeferredActorSpawnFromClass` and `UGameplayStatics:FinishSpawningActor` to spawn an actor.  
```
I don't think that this detail needs to be documented here because it's an internal implementation detail that shouldn't affect mod creators.